homeassistant/components/local_ip/sensor.py

```diff
@@ -0,0 +1,34 @@
+"""Sensor platform for local_ip."""
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import get_local_ip
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry, async_add_entities):
+    """Set up the platform from config_entry."""
+    name = config_entry.data["name"]
+    return async_add_entities([IPSensor(name)], True)
```

Comment:
Nothing is checking this return value. Please remove the `return` and just call `async_add_entities`.